pkg/drivers/powerflex.go

```diff
@@ -375,7 +381,7 @@ func ExtractZonesFromSecret(ctx context.Context, kube client.Client, namespace s
 		}
 		err = yaml.Unmarshal(configs, &yamlConfig)
 		if err != nil {
-			return nil, fmt.Errorf("unable to unmarshal multi-array configuration[%v]", err)
+			return nil, fmt.Errorf("unable to unmarshal no Zone or Name or LabelKey found multi-array configuration[%v]", err)
```

Comment:
do we know if there's no zone or name or label key at this part?

Reply:
removed.